cmd/kube-proxy/app/server.go

```diff
@@ -399,6 +399,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 	proxyMode := getProxyMode(string(config.Mode), iptInterface, iptables.LinuxKernelCompatTester{})
 	if proxyMode == proxyModeIPTables {
 		glog.V(0).Info("Using iptables Proxier.")
+		nodeIP := net.ParseIP(config.BindAddress)
+		if local, _ := isLocalIP(nodeIP); !local {
```

Comment:
I swear I commented on this, but I can't see my own comment.  Who cares if it is local or not.  If it is not, it should fail with a useful error anyway.  Just go simpler.  If bind-address is provided, use it.  else do what we did before.